src/main/java/net/sf/jabref/gui/BasePanel.java

```diff
@@ -2179,6 +1886,10 @@ public boolean isUpdatedExternally() {
         return updatedExternally;
     }
 
+    public void setUpdatedExternally(boolean b) {
+        updatedExternally = b;
```

Comment:
Please use a more speaking variable name instead of b, maybe the same name as for the variable